lib/view/dailyTaskView.dart

```diff
@@ -57,10 +57,8 @@ class TaskViewState extends State<TaskView> {
   /// Dummy function called by taskCard, does not need implementation for daily view 
   void toggleCompleted(Task task) {}
 
-  /// A void function that asynchronously selects a date and fetches tasks for that date.
-  Future<void> selectDate() async {
-    DateTime selectedDate =
-        await datePicker(context, initialDate: today) ?? today;
+  /// A void function that takes a date and asynchronously fetches tasks for that date.
+  Future<void> resetView(DateTime selectedDate) async {
```

Comment:
i think taskbar and datepicker and addbuttonform depend on selectdate right?

Reply:
im not sure what this means, but the functionality shouldn't have changed

Reply:
changed a missing clause. every view currently has a resetview function that the topbar calls. dart isnt good at understanding live transitive object calls, but tbh, neither are most languages, but if it existed, the signature should be the same